lib/senscritique.rb

```diff
@@ -13,9 +13,11 @@ def self.scrap_ranking
 
     movies_list.each do |movie|
       ranking = movie.css('.elto-rank-item').text.to_i
-      title = movie.css('.elco-original-title').text
+      title = movie.css('.elco-original-title').text == '' ?
+        movie.css('.elco-title .elco-anchor').text : movie.css('.elco-original-title').text
```

Comment:
```suggestion
      title = movie.css('.elco-original-title').text
      title = movie.css('.elco-title .elco-anchor').text if title.empty?
```
I'm slightly more at ease with this, WDYT?